test/bind.sh

```diff
@@ -36,6 +43,4 @@ curl \
     -H "Accept: application/json" \
     -H "X-Broker-API-Originating-Identity: " \
     -d "$req" \
-    "https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/service_instances/$INSTANCE_ID/service_bindings/$BINDING_ID?accepts_incomplete=true"
-
-
+    "https://asb-openshift-automation-service-broker.$HOSTNAME.nip.io/openshift-automation-service-broker/v2/service_instances/$INSTANCE_ID/service_bindings/$BINDING_ID?accepts_incomplete=true"
```

Comment:
This will only work for `oc cluster up --enable=automation-service-broker`, it won't work in the development use case (ie. after `make deploy`).

Reply:
I'm not sure building out these developer focused scripts dynamically for all our deployment scenarios is in scope. Although to your point, `make deploy` is probably a developer function, so maybe it deserves attention.

Reply:
If they will not work for a developer then -1 to this change. These scripts are PURELY for hooking up to a developers deployment of the broker.